src/features/cart/screen/index.tsx

```diff
@@ -13,6 +15,7 @@ import CartItem from './cartItem';
 import styles from './styles';
 
 const CartScreen = () => {
+  const { navigate } = useNavigation();
```

Comment:
Just to remind you the cart screen already receives the `navigate` via props, but tbh I'm 100% hooks

Reply:
yeah I feel that is much more clean and readable with useNavigation()